cdb2api/cdb2api.h

```diff
@@ -72,7 +72,7 @@ enum cdb2_errors {
 
     CDB2ERR_THREADPOOL_INTERNAL = -20, /* some error in threadpool code */
     CDB2ERR_ANALYZE_ALREADY_RUNNING = -22,
-    CDB2ERR_APPSOCK_LIMIT = -23,
+    // don't add -23
```

Comment:
Maybe add more context to this comment explaining why -23 shouldn't be added right now --- otherwise I worry that people in the future won't understand when its safe to remove this restriction and it will remain indefinitely. What do you think?